variables/_svgs.pcss

```diff
@@ -222,6 +222,18 @@
 	}
 }
 
+/* -----------------------------------------------------------------------------
+ * SVG - Link
+ * ----------------------------------------------------------------------------- */
+
+@svg-load link url('../icons/link.svg') {
```

Comment:
Let's follow color variable names in the svg var name, much like what we've done previously. So we should call this `link-active`. Same with `mail` and `phone` below.